### ISSUE TITLE
Search Nav Widget 

### ISSUE THREAD

**Issue by jbwilmoth** [service-portal/documentation#67]
I've noticed that in the Search Nav widget, line 12 (var sg = GlideRecord('sp_search_group');) is missing the 'new'.

I also added some logic to the html to prevent the panel element from showing if there are no search groups with ng-if="{{data.searchGroups}}" on the last <a> tag.

Not causing me any issues, but thought I'd let you guys know for those that don't clone the widget.

**Comment by CapaJC** [comment]
Thanks, Joe, those are good fixes. 

--J

On Jul 27, 2016, at 3:21 PM, Joe Wilmoth <notifications@github.com> wrote:

I've noticed that in the Search Nav widget, line 12 (var sg = GlideRecord('sp_search_group');) is missing the 'new'.

I also added some logic to the html to prevent the panel element from showing if there are no search groups with ng-if="{{data.searchGroups}}" on the last tag.

Not causing me any issues, but thought I'd let you guys know for those that don't clone the widget.
